src/pages/Trainee/TraineeList.jsx

```diff
@@ -154,10 +144,60 @@ class TraineeList extends React.Component {
   };
 
   componentDidMount = () => {
-    this.setState({ loading: false });
-    this. getTraineeData(); 
+    // this.setState({ loading: false });
+    // this. getTraineeData(); 
+    const {
+      data: {
+        refetch,
+      },
+    } = this.props;
+    const { data: { subscribeToMore } } = this.props;
+    subscribeToMore({
+      document: UPDATED_TRAINEE_SUB,
+      updateQuery: (prev, { subscriptionData }) => {
+        if (!subscriptionData) return prev;
+        const { getAllTrainees: { data } } = prev;
+        const { data: { traineeUpdated } } = subscriptionData;
+        const updatedRecords = [...data].map((records) => {
+          if (records.originalId === traineeUpdated.originalId) {
+            return {
+              ...data,
+              ...traineeUpdated,
+            };
+          }
+          return data;
```

Comment:
The parameter name in the map() callback function is `records`, so it should return `records`  instead of `data`